src/isar/state_machine/state_machine.py

```diff
@@ -263,7 +247,9 @@ def _off(self) -> None:
     def _resume(self) -> None:
         self.logger.info(f"Resuming mission: {self.current_mission.id}")
         self.current_mission.status = MissionStatus.InProgress
+        self.current_mission.error_message = None
```

Comment:
Will this not erase error messages if you pause a mission with error and resume it? That is maybe the point

Reply:
From my point of view I think it makes sense that the error message should be removed if you resume the mission. Then if there are problems related to the whole mission again it will reappear beacuse the same error happens again. Note that individual errors for tasks and steps will not be affected.